src/content/vintage/platform-overview/observability/_index.md

```diff
@@ -17,33 +17,38 @@ user_questions:
   - What app do you recommend for monitoring?
   - What app do you recommend for logging?
   - What app do you recommend for tracing?
-last_review_date: 2022-11-18
+last_review_date: 2024-02-28
 ---
 
-Observability is based on three main data sources: logs, metrics and tracing\*. To cover these needs Giant Swarm provides its customers with a managed observability stack. The apps included in this stack are listed below. We chose these apps based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.
+Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs Giant Swarm provides its customers with a fully managed observability platform provided with 24//7 monitoring and alerting The tools included in the observabily platform are listed below.
 
-**What do we mean by a "managed" app?** We mean offering similar Service Level Objectives as the rest of our offering as described in [The Radical Way Giant Swarm Handles Service Level Objectives](https://www.giantswarm.io/blog/the-radical-way-giant-swarm-handles-service-level-objectives).
+We chose these applications based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.
 
-In brief, super fast response times from engineers, 24//7 monitoring and alerting by us, and in general, having shared goals and responsibilities to keep services up and get them up when they go down.
+If you want to learn more about observability, we recommend that you read [this](https://opentelemetry.io/docs/concepts/observability-primer/)
 
-In addition, we strive to uphold our "30 Day Upgrade Promise," our commitment to keep all managed apps within 30 days of latest upstream releases.
+To summarize, for observability, we recommend and provide these tools:
 
-To summarize, for observability, we recommend and provide these four managed apps:
+## Loki for logging
+
+To store, index and make logs (containers, kubernetes events, audit logs) searchable. You can also install our [Loki App](https://github.com/giantswarm/loki-app/blob/master/README.md) on your clusters.
```

Comment:
```suggestion
To store, index, and make logs (containers, kubernetes events, audit logs) searchable. You can also install our [Loki App](https://github.com/giantswarm/loki-app/blob/master/README.md) on your clusters.
```

Reply:
IIUC, the implied distinction here is that we run Loki on MCs for logs we collect, but customers must also install Loki themselves if they want to use it for their own log collection. Is that right?
If so, that could be more explicit here as I found it slightly confusing

Reply:
Well the implied direction is that we will bé able to store their logs if they want to but they  can also install it by themselves if they want to, but maybe that part should be explained better. I'm not sure how though 😂

Reply:
What is the benefit if they run Loki themselves?